scripts/index-genenetwork

```diff
@@ -529,7 +529,7 @@ def is_data_modified(xapian_directory: str,
             ])
         # Return a zero exit status code when the data has changed;
         # otherwise exit with a 1 exit status code.
-        generif = pathlib.Path("/var/lib/data/generif-metadata.ttl")
+        generif = pathlib.Path("/var/lib/data/")
```

Comment:
I'm a little worried about this hard-coded path... maybe we should pass this in as an argument. Consider this for future commits.